.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea
 /node_modules
-/dist/app
\ No newline at end of file
+/dist/app
+/package-lock.json
```

Comment:
`package-lock.json` is a new npm file, it is required. if you want to know more about it:
https://docs.npmjs.com/files/package-lock.json